nccharles/PropertyPro-lite
# #167038704: user can view all specific properties like 2-bedroom,3-…

### What does this PR do?
Get Specific Properties

### Description of Task to be completed?
 user can view all specific properties like 2-bedroom,3-bedroom, etc...

### How should this be manually tested?
Git clone this repo
`npm install`
`npm run start-dev`
then Open Postman
use this endpoint to get specific properties
`http://localhost:8080/api/v1/property/?type=2-bedroom`
**Or**
>- Just Click on this link to verify online:
https://propertypro-lite-api.herokuapp.com/api/v1/property/?type=2-bedroom
### What are the relevant pivotal tracker stories?
#167038704
### Screenshots (if appropriate)
![Screenshot at 2019-07-12 18-47-13](https://user-images.githubusercontent.com/18301606/61144591-88b40800-a4d5-11e9-90be-cd62876a8cb0.png)
